quickwit/quickwit-indexing/src/actors/publisher.rs

```diff
@@ -147,7 +147,7 @@ impl Handler<SplitsUpdate> for Publisher {
             );
             return Ok(());
         }
-        info!(new_splits=?split_ids, checkpoint_delta=?checkpoint_delta_opt, "publish-new-splits");
+        info!("publish-new-splits");
```

Comment:
Why?

It has actually extremely useful in the past to be able to search a split id to track its lifecycle.
I assume you did this because checkpoints  are getting too large?

Can you just fix their Debug implementation to make sure it gets shortened if it is too large.

Reply:
actually, I just removed these fields because they are also at the span level:
https://github.com/quickwit-oss/quickwit/blob/996849f60a98a54fa49dfa6721f965f62750d4fc/quickwit/quickwit-indexing/src/actors/publisher.rs#L104-L109

Reply:
Good point!